tests/Doctrine/Tests/Common/Util/DebugTest.php

```diff
@@ -121,6 +121,8 @@ public function testDisablesOutput()
      * @param array<string, int> $expected
      *
      * @dataProvider provideAttributesCases
+     *
+     * @requires PHP < 8.1.0
```

Comment:
Somehow PHP 8.1 changed return order of stuff that `Dump` is returning. Given `Dump` is deprecated I'm not going to invest time in fixing its tests for newer PHP versions.

<details>
<summary>For the record, test fails with:</summary>

```
1) Doctrine\Tests\Common\Util\DebugTest::testExportParentAttributes with data set "different-attributes" (Doctrine\Tests\Common\Util\TestAsset\ChildClass Object (...), array(4, 5, 6, 1, 2, 3))
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
 '(
+    [childPublicAttribute] => 4
+    [childProtectedAttribute:protected] => 5
+    [childPrivateAttribute:Doctrine\Tests\Common\Util\TestAsset\ChildClass:private] => 6
     [parentPublicAttribute] => 1
     [parentProtectedAttribute:protected] => 2
     [parentPrivateAttribute:Doctrine\Tests\Common\Util\TestAsset\ParentClass:private] => 3
-    [childPublicAttribute] => 4
-    [childProtectedAttribute:protected] => 5
-    [childPrivateAttribute:Doctrine\Tests\Common\Util\TestAsset\ChildClass:private] => 6
 )
 '

/home/runner/work/common/common/tests/Doctrine/Tests/Common/Util/DebugTest.php:133

2) Doctrine\Tests\Common\Util\DebugTest::testExportParentAttributes with data set "same-attributes" (Doctrine\Tests\Common\Util\TestAsset\ChildWithSameAttributesClass Object (...), array(4, 5, 6, 3))
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
 '(
     [parentPublicAttribute] => 4
     [parentProtectedAttribute:protected] => 5
+    [parentPrivateAttribute:Doctrine\Tests\Common\Util\TestAsset\ChildWithSameAttributesClass:private] => 6
     [parentPrivateAttribute:Doctrine\Tests\Common\Util\TestAsset\ParentClass:private] => 3
-    [parentPrivateAttribute:Doctrine\Tests\Common\Util\TestAsset\ChildWithSameAttributesClass:private] => 6
 )
 '
```
</details>

Reply:
PHP 8.1 changed the order in which class memebers appear if an object is cast to an array.